src/state/schedule-data/schedule-data.action-creator.ts

```diff
@@ -143,53 +144,30 @@ export class ScheduleDataActionCreator {
   }
 
   static updateSchedule(newScheduleModel: ScheduleDataModel): ScheduleActionModel {
-    // TODO: make separate action creator for Tmp
```

Comment:
Nie do końca rozumiem o co dokładnie tutaj chodziło, ale z racji tego że to nie bolało do tej pory doszedłem do wniosku, że nie ma potrzeby kombinować.